### PR TITLE
Fix/web audit fixes

### PR DESCRIPTION
**Pull request by rschwabco**
* Adds og:image and twitter:image
* Adds `robots.txt` with disallow rules for URLs that shouldn't be crawled
* Updates the title and title delimiter

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -5,7 +5,8 @@ const darkCodeTheme = require("prism-react-renderer/themes/dracula");
 /** @type {import('@docusaurus/types').DocusaurusConfig} */
 (
   module.exports = {
-    title: "Aserto Documentation",
+    title: "Aserto Docs",
+    titleDelimiter: " - ",
     tagline: "Welcome to modern authorization.",
     url: "https://docs.aserto.com",
     baseUrl: "/",
```
Comment (ogazitt) [inline docusaurus.config.js:12]: is this out of date with my change from last night to add line 17: 
`    trailingSlash: false,`

File: docs/api-reference.mdx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_label: API Reference
+sidebar_label: API Reference Yay!
```
Comment (ogazitt) [inline docs/api-reference.mdx]: Yay for enthusiasm, but do we want a Yay here? :)

Comment (rschwabco) [reply docs/api-reference.mdx]: lol I was showing Noa how to commit, somehow this ended here. removed.

File: docusaurus.config.js (hunk 1 above)
Comment (rschwabco) [reply docusaurus.config.js:12]: I merged main into this branch today.

File: docs/api-reference.mdx (hunk 2 above)
Comment (ogazitt) [reply docs/api-reference.mdx]: :)

**Review by ogazitt — APPROVED** [review]

**Pull request merged**